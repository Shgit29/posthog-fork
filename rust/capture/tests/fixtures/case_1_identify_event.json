{
  "title": "posthog-js identify event",
  "path": "/e/?ip=1&_=1694769302325&ver=1.78.5",
  "method": "POST",
  "content_encoding": "",
  "content_type": "application/x-www-form-urlencoded",
  "ip": "127.0.0.1",
  "now": "2023-09-15T09:15:02.328551+00:00",
  "historical_migration": false
}