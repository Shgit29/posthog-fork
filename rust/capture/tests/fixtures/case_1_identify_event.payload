data=eJ1FWlkIjoiMDE4YTk4MWYtNGIzNS03YzY2LTgyNWUtOGIwZWVhZTY1NzE0IiwiZXZlbnQiOiIkdXNlciIsInByb3BlcnRpZXMiOnsiJG9zIjoiTWFjIE9TIFgiLCIkb3NfdmVyc2lvbiI6IjEwLjE1LjAiLCIkYnJvd3NlciI6IkZpcmVmb3giLCIkZGV2aWNlX3R5cGUiOiJEZXNrdG9wIiwiJGN1cnJlbnRfdXJsIjoiaHR0cDovL2xvY2FsaG9zdDo4MDAwLyIsIiRob3N0IjoibG9jYWxob3N0OjgwMDAiLCIkcGF0aG5hbWUiOiIvIiwiJGJyb3dzZXJfdmVyc2lvbiI6MTE3LCIkYnJvd3Nlcl9sYW5ndWFnZSI6ImVuLVVTIiwiJHNjcmVlbl9oZWlnaHQiOjEwNTIsIiRzY3JlZW5fd2lkdGgiOjE2MjAsIiR2aWV3cG9ydF9oZWlnaHQiOjkxMSwiJHZpZXdwb3J0X3dpZHRoIjoxNTQ4LCIkbGliIjoid2ViIiwiJGxpYl92ZXJzaW9uIjoiMS43OC41IiwiJGluc2VydF9pZCI6IjQwcHQxdWNwazc0NmlsZXAiLCIkdGltZSI6MTY5NDc2OTMwMi4zMjUsImRpc3RpbmN0X2lkIjoicFdBazFUeVFRM0NONzMzMlVxUXhuSDZwM0ZWOE5KZDdwWTY0N0VkTmt4V2MiLCIkZGV2aWNlX2lkIjoiMDE4YTk4MWYtNGIyZC03OGFlLTk0N2ItYmVkYmFhMDJhMGY0IiwiJHVzZXJfaWQiOiJwV0FrSVR5UTNDTjczMzJVcVF4bkg2cDNGVjhOSmQ3cFk2NDdFZG5reFdjIiwiJHJlZmVycmVyIjoiJGRpcmVjdCIsIiRyZWZlcnJpbmdfZG9tYWluIjoiJGRpcmVjdCIsIiRhbm9uX2Rpc3RpbmN0X2lkIjoiMDE4YTk4MWYtNGIyZC03OGFlLTk0N2ItYmVkYmFhMDJhMGY0IiwidG9rZW4iOiJwaGNfcWdVRnA1d3oyQmpDNFJzRmpNR3BRM1BHUnNpNlEwQ0cwQ1A0TkFhYzBJIiwiJHNlc3Npb25faWQiOiIwMThhOTgxZi00YjJlLTdiZDMtYmIzMC02NjE3YmY0ODg4NjkiLCIkd2luZG93X2lkIjoiMDE4YTk4MWYtNGIyZS03YmQzLWJiMzAtNjYxODEwZmVkNjVmIn0sIiRzZXQiOnt9LCIkc2V0X29uY2UiOnt9LCJ0aW1lc3RhbXAiOiIyMDIzLTA5LTE1VDA5OjE1OjAyLjMyNVoiLCJkaXN0aW5jdF9pZCI6InBXQWtJVHlRUzNDTjczMzJVcVF4bkg2cDNGVjhOSmQ3cFk2NDdFZG5reFdjIiwiJGRldmljZV9pZCI6IjAxOGE5ODFmLTRiMmQtNzhhZS05NDdiLWJlZGJhYTAyYTBmNCIsIiR1c2VyX2lkIjoicFdBazFUeVFRM0NONzMzMlVxUXhuSDZwM0ZWOE5KZDdwWTY0N0VkTmt4V2MiLCIkcmVmZXJyZXIiOiIkZGlyZWN0IiwiJHJlZmVycmluZ19kb21haW4iOiIkZGlyZWN0IiwiJGFub25fZGlzdGluY3RfaWQiOiIwMThhOTgxZi00YjJkLTc4YWUtOTQ3Yi1iZWRiYWEwMmEwZjQiLCJ0b2tlbiI6InBoY19xZ1VGcDV3ejJCakM0UnNGak1HcFEzUEdSc2k2UTBDRzBDUDROQWFjMEkiLCIkc2Vzc2lvbl9pZCI6IjAxOGE5ODFmLTRiMmUtN2JkMy1iYjMwLTY2MTdiZjQ4ODg2OSIsIiR3aW5kb3dfaWQiOiIwMThhOTgxZi00YjJlLTdiZDMtYmIzMC02NjE4MTBmZWQ2NWYifQ%3D%3D