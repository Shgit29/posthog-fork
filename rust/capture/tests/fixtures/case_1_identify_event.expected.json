[
  {
    "uuid": "018a981f-4b35-7c66-825e-8b0eeae65714",
    "distinct_id": "pWAkITyQ3CN7332UqQxnH6p3FV8VJd7pY647EdNkxWc",
    "ip": "127.0.0.1",
    "site_url": "http://localhost:8000",
    "data": "{\"uuid\": \"018a981f-4b35-7c66-825e-8b0eeae65714\", \"event\": \"$identify\", \"properties\": {\"$os\": \"Mac OS X\", \"$os_version\": \"10.15.0\", \"$browser\": \"Firefox\", \"$device_type\": \"Desktop\", \"$current_url\": \"http://localhost:8000/\", \"$host\": \"localhost:8000\", \"$pathname\": \"/\", \"$browser_version\": 117, \"$browser_language\": \"en-US\", \"$screen_height\": 1052, \"$screen_width\": 1620, \"$viewport_height\": 911, \"$viewport_width\": 1548, \"$lib\": \"web\", \"$lib_version\": \"1.78.5\", \"$insert_id\": \"40pt1ucpk746ilep\", \"$time\": 1694769302.325, \"distinct_id\": \"pWAkITyQ3CN7332UqQxnH6p3FV8VJd7pY647EdNkxWc\", \"$device_id\": \"018a981f-4b2d-78ae-947b-bedbaa02a0f4\", \"$user_id\": \"pWAkITyQ3CN7332UqQxnH6p3FV8VJd7pY647EdNkxWc\", \"$referrer\": \"$direct\", \"$referring_domain\": \"$direct\", \"$anon_distinct_id\": \"018a981f-4b2d-78ae-947b-bedbaa02a0f4\", \"token\": \"phc_qgUFp5wz2BjC4RsFjMGpQ3PGDrsi6Q0CG0CP4NAac0I\", \"$session_id\": \"018a981f-4b2e-7bd3-bb30-6617bf488869\", \"$window_id\": \"018a981f-4b2e-7bd3-bb30-661810fed65f\"}, \"$set\": {}, \"$set_once\": {}, \"timestamp\": \"2023-09-15T09:15:02.325Z\"}",
    "now": "2023-09-15T09:15:02.328551+00:00",
    "sent_at": "2023-09-15T09:15:02.325000+00:00",
    "token": "phc_qgUFp5wz2BjC4RsFjMGpQ3PGDrsi6Q0CG0CP4NAac0I"
  }
]